File: Tests/installation_tests/swift_package_manager/with_objc/SPMTest/ViewController.m
//
//  ViewController.m
//  CocoapodsTest
//
//  Created by Ben Guo on 3/20/18.
//  Copyright © 2018 Stripe. All rights reserved.
//

#import "ViewController.h"
@import Stripe;

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [StripeAPI setDefaultPublishableKey:@"foo"];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: Tests/installation_tests/cocoapods/with_frameworks_swift/CocoapodsTest/ViewController.swift
//
//  ViewController.swift
//  CocoapodsTest
//
//  Created by Jack Flintermann on 8/4/15.
//  Copyright (c) 2015 jflinter. All rights reserved.
//

import Stripe
import StripeApplePay
import StripeCardScan
@_spi(PrivateBetaConnect) import StripeConnect
import StripeFinancialConnections
import StripeIdentity
import StripePaymentSheet
import StripeIssuing
import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        StripeAPI.defaultPublishableKey = "test"

        let _ = IdentityVerificationSheet(verificationSessionClientSecret: "test")

        let _ = FinancialConnectionsSheet(
            financialConnectionsSessionClientSecret: "",
            returnURL: nil
        )

        let _ = CardImageVerificationSheet(
            cardImageVerificationIntentId: "foo",
            cardImageVerificationIntentSecret: "foo"
        )

        let _ = PaymentSheet(
            paymentIntentClientSecret: "",
            configuration: PaymentSheet.Configuration()
        )
        
        let _ = STPPushProvisioningDetailsParams.init(
            cardId: "test",
            certificates: [],
            nonce: Data(),
            nonceSignature: Data()
        )

        let _ = EmbeddedComponentManager {
            nil
        }
    }
}
